runtime/zeitgeist/src/parameters.rs

```diff
@@ -288,6 +288,35 @@ parameter_types! {
     /// Pallet identifier, mainly used for named balance reserves. DO NOT CHANGE.
     pub const TreasuryPalletId: PalletId = PalletId(*b"zge/tsry");
 
+    // Bounties
+    /// The amount held on deposit for placing a bounty proposal.
+    pub const BountyDepositBase : Balance = 10 * BASE;
+    /// The delay period for which a bounty beneficiary need to wait before claim the payout.
+    pub const BountyDepositPayoutDelay : BlockNumber = 3 * BLOCKS_PER_DAY;
+
+    /// Bounty duration in blocks.
+    pub const BountyUpdatePeriod : BlockNumber = 14 * BLOCKS_PER_DAY;
```

Comment:
This constant is relatively low. I am not sure about the exact implications of that. For example, Karura uses 35 days and Kusama and Polkadot 90 days. Maybe 

```suggestion
    pub const BountyUpdatePeriod : BlockNumber = 35 * BLOCKS_PER_DAY;
```
@Boibit

Reply:
> This constant is relatively low. I am not sure about the exact implications of that. For example, Karura uses 35 days and Kusama and Polkadot 90 days. Maybe
> 
> @Boibit

Agreed. 
35 days may be generous but it allows for flexibility and broadens the communication window for review.